lib/media/streaming_engine.js

```diff
@@ -1136,14 +1133,16 @@ shaka.media.StreamingEngine = class {
       if (inaccurateTolerance) {
         mediaState.segmentIterator =
             mediaState.stream.segmentIndex.getIteratorForTime(lookupTime);
-        ref = mediaState.segmentIterator.next().value;
+        ref = mediaState.segmentIterator &&
+            mediaState.segmentIterator.next().value;
       }
-      if (!ref) {
+      if (! ref) {
```

Comment:
Please remove this extra space between `!` and `ref` to conform to the style of the rest of the codebase.